app/lib/screens/wallets/send.dart

```diff
@@ -114,6 +114,15 @@ class _WalletSendScreenState extends State<WalletSendScreen> {
         toAddressError = 'Invaild Stellar address';
         return false;
       }
+
+      final matchingWallets = widget.allWallets
+          .where((wallet) => wallet.stellarAddress == toAddress);
+      final Wallet? wallet =
+          matchingWallets.isNotEmpty ? matchingWallets.first : null;
+      if (wallet != null && wallet.stellarBalance == '-1') {
+        toAddressError = 'Wallet not activated on stellar';
+        return false;
+      }

```

Comment:
```suggestion
      final wallet = widget.allWallets
          .where((wallet) => wallet.stellarAddress == toAddress).toList().first;
      if (wallet.stellarBalance == '-1') {
        toAddressError = 'Wallet not activated on stellar';
        return false;
      }
```

Reply:
In the provided code if there's no matching wallet, flutter will throw an exception.
That's why i handled it that way to check first there's a matching wallet.